hack/lint-paths.txt

```diff
@@ -0,0 +1,16 @@
+pkg/instancetype
+pkg/libvmi
+pkg/network/admitter
+pkg/network/namescheme
+pkg/network/domainspec
+pkg/network/deviceinfo
+pkg/virtctl/credentials
```

Comment:
There are missing paths here which have not been moved from the original.

Reply:
done